R/formatting.R

```diff
@@ -168,7 +168,7 @@ fmt_corr <- function(x, digits, output = NULL) {
 
 #' @export
 #' @rdname formatting
-fmt_prop <- function(x, digits, fmt_small = TRUE, keep_zero = FALSE) {
+fmt_prop <- function(x, digits, fmt_small = TRUE, keep_zero = FALSE, output) {
```

Comment:
output should have a default value. Probably `output = NULL` like in the `fmt_corr()` and `fmt_minus()` functions.

Should also check output using `check_output()`